asab/metrics/metrics.py

```diff
@@ -110,11 +110,14 @@ class EPSCounter(Counter):
 	"""
 
 	def __init__(self, init_values=None):
+		if init_values:
+			for v in init_values.values():
+				assert isinstance(v, (int, float))
```

Comment:
You don't need that assert here, the `float(v)` conversion will deal with then in the next step IMHO.

Reply:
My thoughts... Nothing prevents you from passing string or bool as values (and Influx also takes string and bool values, Prometheus doesn't). But ok, let's assume the user is not an idiot and understands that when the `float(v)` expression fails.